framework/jquery/validators/client/BooleanValidator.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\jquery\validators\client;
+
+use yii\jquery\ValidationAsset;
+use yii\validators\client\ClientValidator;
+
+/**
+ * BooleanValidator composes client-side validation code from [[\yii\validators\BooleanValidator]].
+ *
+ * @see \yii\validators\BooleanValidator
+ * @see ValidationAsset
+ *
+ * @author Paul Klimov <klimov.paul@gmail.com>
+ * @since 2.1.0
+ */
+class BooleanValidator extends ClientValidator
+{
+    /**
+     * @inheritdoc
+     */
+    public function build($validator, $model, $attribute, $view)
```

Comment:
It could not be build with any other validator than `BooleanValidator` from core framework so it makes sense to type-hint here and in similar cases.